setup.py

```diff
@@ -26,7 +26,7 @@
 setup(
     # Self-descriptive entries which should always be present
     name='transport_analysis',
-    author='Xu Hong Chen','Hugo MacDermott-Opeskin','Orion Cohen'
+    author='Xu Hong Chen',
```

Comment:
Yeah unfortunately `setup` doesn't support lists of strings. The only workaround is to make the whole thing one string, i.e. `'Xu Hong Chen, Hugo MacDermott-Opeskin, Orion Cohen'`, but it's probably easier to just keep `AUTHORS.md` up-to-date.